src/models/GitHubAuth.js

```diff
@@ -1,46 +1,25 @@
-'use strict'
-
 import keytar from 'keytar'
-import ElectronConfig from 'electron-config'
-
-const isTest = process.env.NODE_ENV === 'test'
-const storage = isTest ? new ElectronConfig({ name: 'config-test' }) : null
 
 const SERVICE = 'gh-notifications-snoozer'
 const ACCOUNT = 'github'
-const KEY = 'token'
 
-class GitHubAuth {
+module.exports = class GitHubAuth {
   static isAuthenticated() {
-    if (!isTest) {
-      return this.getToken() !== null
-    }
-    return storage.has(KEY)
```

Comment:
I too had a pickle of a time. I was able to get it working by mocking it out in tests. I thought that approach made more sense because it keeps test code out of the app.